workflow/rules/star_fusion.smk

```diff
@@ -0,0 +1,51 @@
+# vim: syntax=python tabstop=4 expandtab
```

Comment:
I still think these syntaxing comments are misplaced as they are very specific for one user and should not be part of the code base that is shared among different users.